backend/src/test/java/codezap/likes/service/LikesServiceTest.java

```diff
@@ -0,0 +1,127 @@
+package codezap.likes.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import codezap.fixture.CategoryFixture;
+import codezap.fixture.MemberFixture;
+import codezap.fixture.TemplateFixture;
+import codezap.global.DatabaseIsolation;
+import codezap.global.ServiceTest;
+import codezap.member.domain.Member;
+import codezap.member.dto.MemberDto;
+import codezap.template.domain.Template;
+
+@SpringBootTest
+@DatabaseIsolation
+class LikesServiceTest extends ServiceTest {
+
+    @Autowired
+    private LikesService likesService;
+
+    @Nested
+    @DisplayName("좋아요 기능 테스트")
```

Comment:
기능 테스트라는 말은 없어도 괜찮지 않을까요~!

```suggestion
    @DisplayName("좋아요")
```